### PR TITLE
Clang-format sources

### PR DESCRIPTION
**Pull request by janisozaur**
formats all the sources in the repository.

It's been a while (over a year) since I tried. It seems there are far less changes now.

### CODE REVIEW

**Comment by janisozaur** [comment]
This has been open for over two weeks now, can someone please review/merge?

File: src/CalcViewModel/Common/CopyPasteManager.cpp [hunk 1]
```diff
@@ -69,7 +69,8 @@ void CopyPasteManager::CopyToClipboard(String ^ stringToCopy)
     Clipboard::SetContent(dataPackage);
 }
 
-IAsyncOperation<String ^> ^ CopyPasteManager::GetStringToPaste(ViewMode mode, CategoryGroupType modeType, NumberBase programmerNumberBase, BitLength bitLengthType)
+IAsyncOperation<
```
Comment (tian-lt) [inline src/CalcViewModel/Common/CopyPasteManager.cpp:72]: Seems clang-format is not able to help arrange C++/CX code in the way we'd like to see.
For example, clang-format might have treated `^` as a XOR operator and get confused here.

**Comment by tian-lt** [comment]
Thank you for proposing this PR, @janisozaur.

It's a great idea to use clang-format to keep the code style clean.
However, the codebase of Calculator is mostly written in C++/CX, which I don't think clang-format can handle properly(like the comment above).

It would be great if we narrow down the automation scope to CalcManager and GraphingImpl projects.

**Comment by janisozaur** [comment]
Ok, I will update the scope of this pr

**Comment by tian-lt** [comment]
That sounds great!

BTW, I couldn't find clang-format and its configuration in this PR. So, looks like you formatted the code locally, correct?
If that is true, then maybe me would prefer the way that can help continuously format the codebase in both developing environment and CI pipeline.
What do you think? @joseartrivera 😀

**Comment by janisozaur** [comment]
Clang-format configuration is already present in the repository: https://github.com/microsoft/calculator/blob/master/.clang-format

Regarding CI: see my previous PRs: https://github.com/microsoft/calculator/pull/406 and https://github.com/microsoft/calculator/pull/405

**Comment by Chips1234** [comment]
Can we get a review for this PR, please?

**Comment by janisozaur** [comment]
I've updated this now to fix the C++/CX issues. It is limited mostly to the part discussed earlier, but I took an artistic license to include parts in other areas that fix various misalignments, misindentations and other minor mishaps to better align with rest of code.

**Review by tian-lt — APPROVED** [review]

**Comment by tian-lt** [comment]
Thanks for submitting the updates, it's awesome that C++/CX codebase can be formatted in such good result.
What is the artistic  license you took?

**Comment by janisozaur** [comment]
> What is the artistic license you took?

To omit some of the changes that were misunderstood by clang-format, e.g.:

```diff
diff --git a/src/CalcViewModel/Common/CopyPasteManager.cpp b/src/CalcViewModel/Common/CopyPasteManager.cpp
index 8f84eea..3e6d92d 100644
--- a/src/CalcViewModel/Common/CopyPasteManager.cpp
+++ b/src/CalcViewModel/Common/CopyPasteManager.cpp
@@ -69,7 +69,8 @@ void CopyPasteManager::CopyToClipboard(String ^ stringToCopy)
     Clipboard::SetContent(dataPackage);
 }
 
-IAsyncOperation<String ^> ^ CopyPasteManager::GetStringToPaste(ViewMode mode, CategoryGroupType modeType, NumberBase programmerNumberBase, BitLength bitLengthType)
+IAsyncOperation<
+    String ^> ^ CopyPasteManager::GetStringToPaste(ViewMode mode, CategoryGroupType modeType, NumberBase programmerNumberBase, BitLength bitLengthType)
 {
     // Retrieve the text in the clipboard
     auto dataPackageView = Clipboard::GetContent();
```
or
```diff
diff --git a/src/CalcViewModel/Common/LocalizationService.h b/src/CalcViewModel/Common/LocalizationService.h
index d692137..46e4956 100644
--- a/src/CalcViewModel/Common/LocalizationService.h
+++ b/src/CalcViewModel/Common/LocalizationService.h
@@ -30,8 +30,7 @@ namespace CalculatorApp
             DEPENDENCY_PROPERTY_ATTACHED_WITH_DEFAULT_AND_CALLBACK(LanguageFontType, FontType, LanguageFontType::UITe
xt);
             DEPENDENCY_PROPERTY_ATTACHED_WITH_CALLBACK(double, FontSize);
 
-        internal:
-            static LocalizationService ^ GetInstance();
+            internal : static LocalizationService ^ GetInstance();
             static void OverrideWithLanguage(_In_ const wchar_t* const language);
 
             Windows::UI::Xaml::FlowDirection GetFlowDirection();
```

**Comment by tian-lt** [comment]
Manually correct those wrong formatted statements?

**Comment by janisozaur** [comment]
Just not included them in this commit - manually verified which hunks to include.

**Pull request merged**